src/main/java/com/mycompany/app/App.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2019 Esri
+ * Copyright 2021 Esri
```

Comment:
we don't usually update copyright for changes to an existing source code file: that's the guidance I've had in the past from both samples and doc team. We should only change this if it's brand new code.